src/components/logo.tsx

```diff
@@ -1,11 +1,12 @@
-import { Trans } from "@lingui/macro";
+import { t, Trans } from "@lingui/macro";
 
 export const LogoMobile = () => (
   <svg width={30} height={34}>
     <title id="logo">
-      <Trans id="logo.swiss.confederation">
-        Logo of the Swiss Confederation
-      </Trans>
+      {t({
+        id: "logo.swiss.confederation",
+        message: "Logo of the Swiss Confederation",
+      })}
```

Comment:
Wondering why it was a problem, and why it was not a problem e.g. in `LogoDesktop` down below? 🤔